### PR TITLE
Add a tuples primer

### PR DESCRIPTION
**Pull request by mppf**
I noticed that tuples aren't documented in the Users Guide and that they didn't have a primer.
The closest thing to it was a section in learnChapelInYMinutes.

This PR adds a new primer describing tuples.

Reviewed by @ben-albrecht - thanks!

### CODE REVIEW

File: test/release/examples/primers/tuples.chpl [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+// Getting Started with Tuples
+// ***************************
+// 
+// The easiest way to create a tuple is with a tuple literal.
+// Tuple literals syntax looks like this:
+
+var myTuple = (1, "two");
+
+// Now myTuple stores a tuple with two elements:
```
Comment (ben-albrecht) [inline test/release/examples/primers/tuples.chpl]:
```
``myTuple``
```

File: test/release/examples/primers/tuples.chpl [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+// We can assign to a component in our tuple variable
+myTuple(2) = "four";
+
+// We can print out the tuple. The output format mirrors
+// the Chapel syntax.
+writeln(myTuple); // output: (1, four)
+
+// Sometimes it's useful in generic code to query the size of a tuple.
+// The .size method returns the size of a tuple.
```
Comment (ben-albrecht) [inline test/release/examples/primers/tuples.chpl]:
```
``.size``
```

File: test/release/examples/primers/tuples.chpl [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+var otherTuple: (int, string);
+
+// We can assign to that tuple from a variable with the same type:
+otherTuple = myTuple;
+
+// The following line declares a tuple variable holding 3 real values:
+var threeReals: 3*real;
+writeln(threeReals); // output: (0.0, 0.0, 0.0)
+
+// Tuples that have only one component type support
```
Comment (ben-albrecht) [inline test/release/examples/primers/tuples.chpl]: maybe add: 

> (known as a homogenous tuple)

File: test/release/examples/primers/tuples.chpl [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+// ***************
+
+// Tuples with only one component type can be iterated over using a for loop:
+var sum = 0.0;
+for r in threeReals {
+  sum += r;
+}
+writeln(sum);
+
+// Want to iterate over a tuple with multiple component types?
```
Comment (ben-albrecht) [inline test/release/examples/primers/tuples.chpl]: The rhetorical question feels out of place, being the only question. Maybe just say "To iterate over a tuple with multiple component types, use a `for param` loop.

File: test/release/examples/primers/tuples.chpl [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+// by the compiler so the loop body can use different types in
+// different iterations.
+for param i in 1..myTuple.size {
+  writeln("myTuple(", i, ") = ", myTuple(i));
+}
+
+// Tuple Destructuring
+// *******************
+// 
+// Tuple values can be pulled apart variables during
```
Comment (ben-albrecht) [inline test/release/examples/primers/tuples.chpl]: I prefer the terminology of "can be unpacked" over "can be pulled apart", but I'm not sure if one is more correct.

File: test/release/examples/primers/tuples.chpl [hunk 6]
```diff
@@ -74,18 +77,18 @@ for r in threeReals {
...
   writeln("myTuple(", i, ") = ", myTuple(i));
 }
 
-// Tuple Destructuring
-// *******************
+// Tuple Unpacking
+// ***************
 // 
-// Tuple values can be pulled apart variables during
+// Tuple values can be unpacked apart variables during
```
Comment (ben-albrecht) [inline test/release/examples/primers/tuples.chpl]: into variables? as variables?

I must have missed that before.

File: test/release/examples/primers/tuples.chpl (hunk 3 above)
Comment (cassella) [reply test/release/examples/primers/tuples.chpl]: "homogeneous" (with a second e) would save a typo fix pr later.

File: test/release/examples/primers/tuples.chpl [hunk 7]
```diff
@@ -74,18 +77,18 @@ for r in threeReals {
...
 // variable initialization:
```
Comment (cassella) [inline test/release/examples/primers/tuples.chpl]: This sentence doesn't scan for me.

**Pull request merged**